LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2020 DFINITY Stiftung, ALL RIGHTS RESERVED
```

Comment:
This license had been written for a different repo. Repurposing it here.

Reply:
Not a lawyer, but in my experience if you want to use this kind of a license _and_ accept contributions you need to make every contributor sign a CLA (Contributer License Agreement).